src/settings.c

```diff
@@ -41,6 +41,14 @@ static void on_sidebar_pos_left_changed(GSettings *settings, gchar *key, gpointe
 }
 
 
+static void on_editor_font_changed(GSettings *settings, gchar *key, gpointer user_data)
+{
```

Comment:
The `editor_font` and `symbols_font` do the same thing, so they could be handled the way I proposed above and use the same callback.  There can be different callbacks, its not a choice between one callback and one per setting.

And if you are going to select stuff, at least make it a switch on an enum, not a series of strcmps :)